game.cpp

```diff
@@ -1205,19 +1211,32 @@ void Game::WriteReport()
 
 	forlist(&factions) {
 		Faction *fac = (Faction *) elem;
-		AString str = "report.";
-		str = str + fac->num;
+		string str = "report." + to_string(fac->num);
 
-		if (!fac->IsNPC() ||
-				((((month == 0) && (year == 1)) || Globals->GM_REPORT) &&
```

Comment:
this conditional of '(turn one || gm report) && fac == 1' was tested in a bunch of places, so I made a nice little inline function and use it here and wherever else I saw it.